### PR TITLE
chore(m365): add outlook channel prompt in F5 flow for bot app

### PR DESCRIPTION
**Pull request by kuojianlu**
For launch page:
image
For messaging extension:
image

### CODE REVIEW

File: packages/vscode-extension/src/debug/commonUtils.ts [hunk 1]
```diff
@@ -238,6 +239,25 @@ export async function getLocalTeamsAppId(): Promise<string | undefined> {
   }
 }
 
+export async function getLocalBotId(): Promise<string | undefined> {
+  try {
+    if (isConfigUnifyEnabled()) {
```
Comment (swatDong) [inline packages/vscode-extension/src/debug/commonUtils.ts:244]: Do we have the case that an M365 app with unify config disabled?

**Comment by swatDong** [comment]
Since currently hard to detect outlook channel, do we have any expected error to give some hints to user? E.g., if you see XXX error/behavior on outlook, that means you have not connected your bot to outlook channel yet.

File: packages/vscode-extension/src/debug/commonUtils.ts (hunk 1 above)
Comment (kuojianlu) [reply packages/vscode-extension/src/debug/commonUtils.ts:244]: At current stage, unify config is disabled by default, so M365 feature also needs to check this config.

**Review by swatDong — APPROVED** [review]

**Comment by kuojianlu** [comment]
> Since currently hard to detect outlook channel, do we have any expected error to give some hints to user? E.g., if you see XXX error/behavior on outlook, that means you have not connected your bot to outlook channel yet.

Good idea to give some hints to user. If not connected to Outlook channel, the messaging extension will show "Something went wrong". Maybe we can put this information in README or help doc.
image

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Pull request merged**